copy-fixtures.js

```diff
@@ -0,0 +1,31 @@
+import { promisify } from "node:util";
+import chip from "node:child_process";
+import fs from "node:fs";
+import path from "node:path";
+
+const exec = promisify( chip.exec );
+
+await exec( "npm install --no-save jquery" );
```

Comment:
Why not make it a `devDependency` instead?

Reply:
Yes, this should be fine. Cyclic devDeps should not be an issue, devDeps always are only installed on the top level.